pkg/virt-handler/container-disk/mount.go

```diff
@@ -459,6 +447,24 @@ func (m *mounter) mountKernelArtifacts(vmi *v1.VirtualMachineInstance, verify bo
 		return err
 	}
 
+	record := vmiMountTargetRecord{
+		MountTargetEntries: []vmiMountTargetEntry{
+			{
+				TargetFile: unsafepath.UnsafeAbsolute(targetInitrdPath.Raw()),
```

Comment:
Is it needed to add the entry if initrd path is not provided? I am referring to the PR https://github.com/kubevirt/kubevirt/pull/8209 which aims to solve the issue with kernel-only boot.

Reply:
It is not needed. I am aware of the PR and was relying on the fact that one of these will get merge conflict. I can rebase upon #8209 to make it easier.